Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,11 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
```

Comment:
When `poured_from_bottle` is missing from `INSTALL_RECEIPT.json`. This line is also found in the `used_options` block below. I suppose it's for backward compatibility.

> I think it would be best to make this an else clause in the above case statement.

Indeed, will fix.